.nf-core.yml

```diff
@@ -1,10 +1,22 @@
-repository_type: pipeline
-nf_core_version: "2.14.1"
-
+bump_version: null
 lint:
   files_unchanged:
     - lib/NfcoreTemplate.groovy
   nextflow_config:
     - config_defaults:
         - params.phix_reference
         - params.lambda_reference
+nf_core_version: 3.0.2
+org_path: null
+repository_type: pipeline
+template:
+  author: "Hadrien Gourlé, Daniel Straub, Sabrina Krakau, James A. Fellows Yates, Maxime Borry"
+  description: Assembly, binning and annotation of metagenomes
+  force: false
+  is_nfcore: true
+  name: mag
+  org: nf-core
+  outdir: .
+  skip_features: null
+  version: 3.1.1dev
```

Comment:
```suggestion
  version: 3.2.0
```
I assume?

Reply:
@jfy133 seems to be the same issue as in [detaxizer release PR](https://github.com/nf-core/detaxizer/pull/45)? Did you use `nf-core pipelines bump-version`?

Reply:
Huh yeah... I did do that @jannikseidelQBiC maybe tools doesn't bump that number?